### DISCUSSION TITLE
Custom URL path for ## text

### DISCUSSION THREAD

**Discussion by sheldygg** [squidfunk/mkdocs-material#6312] [Q&A] [answered]
Hello, how i can set my custom path for toc?
It created automaticaly and i have some problems with it.
If my text is all Cyrillic, path was be #_somesumber.
But if my text is all Cyrillic but it contains english alphabet, path was be from english word

**Comment by sisp** [comment, marked as answer]
You can set a custom heading ID using `{ #your-custom-id }` like

https://github.com/squidfunk/mkdocs-material/blob/06ef698ab25163dd056fd7ebc9d8918d3be9651f/docs/getting-started.md?plain=1#L21

using the [`attr_list`](https://python-markdown.github.io/extensions/attr_list/) Markdown extension that you can enable like this:

https://github.com/squidfunk/mkdocs-material/blob/06ef698ab25163dd056fd7ebc9d8918d3be9651f/mkdocs.yml#L123
